File: java/figuras/src/main/java/com/maicol/figuras/rectangle.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.maicol.figuras;

/**
 *
 * @author Maicol
 */
public class rectangle extends figureGeometric{
    private double sideOne;
    private double sideTwo;

    public rectangle(double sideOne, double sideTwo) {
        super("Rectangle", "Rectángulo");
        this.sideOne = sideOne;
        this.sideTwo = sideTwo;
    }

    @Override
    public double calculateArea() {
        return sideOne * sideTwo;
    }
}

File: php/examples/figuras/php/rectangle.php
<?php
class rectangle extends  FigureGeometric{

    private float $sideOne;
    private float $sideTwo;

    public function __construct(float $sideOne, float $sideTwo){
        parent:: __construct("Rectangle","Rectangúlo");
            $this->sideOne = $sideOne;
            $this->sideTwo= $sideTwo;
        }
            public function calculateArea(): float{
                return $this->sideOne*$this->sideTwo;
            }       
        }
?> 